app.js

```diff
@@ -0,0 +1,92 @@
+const comediansList = [
+  {
+    name: 'Ali Wong',
+    photoUrl: 'Ali_Wong.jpeg',
+    description: 'One of the best comedians in the US',
+    background: 'Ali Wong was born and raised in the US in the state of Texas',
+  },
+  {
+    name: 'Dave Chapelle',
+    photoUrl: 'Dave_Chapelle.jpeg',
+    description: 'His fans call him the goat because he is a legend',
+    background: 'Born in ohio, Dave is known all over the world for his comedy',
+  },
+  {
+    name: 'Ali Wong',
+    photoUrl: 'Ali_Wong.jpeg',
+    description: 'One of the best comedians in the US',
+    background: 'Ali Wong was born and raised in the US in the state of Texas',
+  },
+  {
+    name: 'Dave Chapelle',
+    photoUrl: 'Dave_Chapelle.jpeg',
+    description: 'His fans call him the goat because he is a legend',
+    background: 'Born in ohio, Dave is known all over the world for his comedy',
+  },
+  {
+    name: 'Ali Wong',
+    photoUrl: 'Ali_Wong.jpeg',
+    description: 'One of the best comedians in the US',
+    background: 'Ali Wong was born and raised in the US in the state of Texas',
+  },
+  {
+    name: 'Dave Chapelle',
+    photoUrl: 'Dave_Chapelle.jpeg',
+    description: 'His fans call him the goat because he is a legend',
+    background: 'Born in ohio, Dave is known all over the world for his comedy',
+  },
+];
+
+const comediansListElement = document.getElementById('comedians-list');
+
+if (document.title === 'About') {
+  document.getElementById('about-link').classList.add('active');
+}
+
+const addComediansToList = (amountDisplayed, amountToDisplay) => {
+  for (let i = amountDisplayed; i < amountToDisplay; i += 1) {
+    const comedian = comediansList[i];
+    const comedianElement = document.createElement('li');
+    comedianElement.classList.add('comedian');
+    comedianElement.innerHTML = `
+      <div class='photo'><img src='media/${comedian.photoUrl}'></div>
+      <div class='information'>
+        <h3 class='name'>${comedian.name}</h3>
+        <p class='description'>${comedian.description}</p>
+        <hr>
+        <p class='background'>${comedian.background}</p>
+      </div>
+      `;
```

Comment:
I like the way you implemented the javascript for generating the speakers. But it seems you can improve the design by following the original design idea. So for this please make sure to add the background picture for the speaker image, and also increase its size a little bit to make it look much more similar to the one in the original design..

![image](https://user-images.githubusercontent.com/34779928/203857787-c56558b3-197e-4e48-b192-184e6b5fb880.png)